README.md

```diff
@@ -2,7 +2,20 @@
 
 Use Docker for machine learning, without all the pain.
 
-Cog gives you a consistent environment to run your model in – for developing on your laptop, training on GPU machines, and for other people working on the model. Then, when the model is trained and you want to share or deploy it, you can bake the model into a Docker image that serves a standard HTTP API.
+- [What is Cog?](#what-is-cog)
```

Comment:
This top bit is intended to be a bit like a landing page for the project, not documentation. "Develop and train in a consistent environment", "Put a trained model in a Docker image", and "Why are we building this?" are all things you might imagine being on the home page of the Cog website, were it to exist.

I wonder whether the table of contents should live under those bits, so we aren't burying it? Maybe we could change that if/when we have a website.

Reply:
I get that, but I think having a TOC up top is useful because you can see at a glance what's covered. I've always found it a little weird that the installation and usage docs are buried so deep in the page. Maybe it would look a little tighter if all the landing-page-ish content was tucked under the one "What is Cog?" TOC link?

- [What is Cog?](#what-is-cog)
- [Prerequisites](#prerequisites)
- [Install](#install)
- [Upgrade](#upgrade)
- [Next steps](#next-steps)
- [Need help?](#need-help)
- [Contributors ✨](#contributors-%E2%9C%A8)

Reply:
Yeah, that's better. It still feels like it needs more above the fold though. We can always tweak later -- this is being rewritten in future anyway.